src/nimlsp.nim

```diff
@@ -387,11 +387,13 @@ while true:
                 # Try to guess the size of the identifier
                 let
                   message = diagnostic.nimDocstring
-                  endcolumn = diagnostic.column +  message.rfind('\'') - message.find('\'') - 1
+                  endcolumn = max(diagnostic.column + message.rfind('\'') - message.find('\'') - 1, 0)
```

Comment:
This doesn't really do much as it got max'ed with `diagnostic.column` later anyways. I think the problem here is that things with line or column <0 aren't visible in the document and therefore should be returned in some other way.